techscreenr-angular/models/question-interface.ts

```diff
@@ -0,0 +1,12 @@
+import { Answer } from './answer-interface';
+import { QuestionTypeEnum } from './question-type.enum';
+
+export interface Question {
```

Comment:
@amartinelli12 Each question needs to have a property for storing example code as an array. For example, what if we want to provide example html and example css and want them to answer a question based on the two sets of code (like how would the css make this html look). Any question regardless of type should have an option to provide example code for the question in addition to a separate question description.